editor/src/utils/value-parser-utils.ts

```diff
@@ -325,6 +325,15 @@ export function parseJsx(_: unknown, value: unknown): ParseResult<JSXParsedValue
         name: type,
       })
     }
+    if (value.hasOwnProperty('props')) {
```

Comment:
is this a somewhat unrelated change?

Reply:
Yes, this was just a small issue we found during the more extensive jsx prop testing.
However, after this PR we may rethink on only relying on the runtime value and not on the parsed model in `parseJsx`. But that can come in a subsequent PR.